.github/workflows/build.yml

```
@@ -0,0 +1,25 @@
+name: Continuous Integration
+
+run-name: ${{ github.ref_name }}
+
+on:
+  push:
+    branches:
+      - dev
+      - master
+      - release-*
+      - '[0-9]+.[0-9]+.x'
+    paths-ignore:
+      - '.github/**'
+      - '**/README.md'
+      - '!.github/workflows/build.yml'
+      - '!.github/workflows/_reusable_build.yml'
+
+jobs:
+  call-build-workflow:
```

use a unique name like:

```suggestion
  build-branch:
```

---

Also renamed file similarly to https://github.com/bonitasoft/bonita-studio-sp/commit/f3a65031ae64be8c05308c34cfc947c1b345642e